src/html/components/concept.html

```diff
@@ -32,7 +32,7 @@ <h3>She cheers you on.</h3>
     <div class="messages">
       <ul>
         <li class="messages-left">Hey Mike! Nice job, your cholesterol is down 20%.</li>
-        <li class="messages-left recent">Treat yourself to something nice! 😊</li>
+        <li class="messages-left recent"><span></span>Treat yourself to something nice! 😊</li>
```

Comment:
Maybe add a `class` of `avatar` or `message-avatar` to this. In the context of a larger project, it's helpful to have the classname for identification.